### ISSUE TITLE
When I run apollo.sh, I can't get access to abseil, how to solve this problem 

### ISSUE THREAD

**Issue by markofan** [ApolloAuto/apollo#11320]
When I run apollo.sh(./apollo.sh build_cpu), I can't get access to abseil, how to solve this problem ？

ERROR INFORMATION:
INFO: (05-23 10:25:58.589) Timeout connecting to https://github.com/abseil/abseil-cpp/archive/cd95e71df6eaf8f2a282b1da556c2cf1c9b09207.tar.gz

**Comment by storypku** [comment]
> When I run apollo.sh(./apollo.sh build_cpu), I can't get access to abseil, how to solve this problem ？
> 
> ERROR INFORMATION:
> INFO: (05-23 10:25:58.589) Timeout connecting to https://github.com/abseil/abseil-cpp/archive/cd95e71df6eaf8f2a282b1da556c2cf1c9b09207.tar.gz

On which  branch ?   What's the bazel version (By  running `bazel version`) ？
Generally, u can use a proxy when before running `apollo.sh` by setting like
```
export HTTP_PROXY=agent.example.com:8018
export HTTPS_PROXY=agent.example.com:8018
```

**Comment by markofan** [comment]
> On which branch ? What's the bazel version (By running `bazel version`) ？
> Generally, u can use a proxy when before running `apollo.sh` by setting like
> 
> ```
> export HTTP_PROXY=agent.example.com:8018
> export HTTPS_PROXY=agent.example.com:8018
> ```
I have checked the code, this issue happened when I run this code
"  bazel build $JOB_ARG $DEFINES -c $@ $BUILD_TARGETS" <Line 147 in apollo.sh>
The bazel version is 3.1.0
I have already set the HTTP_PROXY, but it doesn't work.
Can I install "abseil" by myself without by the code?
=================================
我觉得我的问题在于即使设置了代理，bazel也不能从网上下载东西，一开始是不能下载“https://github.com/ApolloAuto/apollo-contrib/releases/download/v3.0.0/plat-sw-3.0.0.1.zip”，我看代码发现写在WORKSPACE.in中，于是自己把这个文件下载，放在/home/tmp中。但是我不知道这个“abseil”安装包是应该放在哪一个文件夹中。这个能够自己先安装吗，不通过指令？

**Comment by storypku** [comment]
> > On which branch ? What's the bazel version (By running `bazel version`) ？
> > Generally, u can use a proxy when before running `apollo.sh` by setting like
> > ```
> > export HTTP_PROXY=agent.example.com:8018
> > export HTTPS_PROXY=agent.example.com:8018
> > ```
> 
> # I have checked the code, this issue happened when I run this code
> " bazel build $JOB_ARG $DEFINES -c $@ $BUILD_TARGETS" <Line 147 in apollo.sh>

As far as I know, only the docker_dev branch has bazel 3.1.0 enabled.

With bazel 3.x, you can place your pre-downloaded tarballs under /apollo/.cache/distdir, and enable the `--distdir=/apollo/.cache/distdir` option when calling bazel build/test. Eg. ` bazel build --distdir=/apollo/.cache/distdir //cyber/... `, or  `bazel test --distdir=/apollo/.cache/distdir`.

**Comment by storypku** [comment]
> When I run apollo.sh(./apollo.sh build_cpu), I can't get access to abseil, how to solve this problem ？
> 
> ERROR INFORMATION:
> INFO: (05-23 10:25:58.589) Timeout connecting to https://github.com/abseil/abseil-cpp/archive/cd95e71df6eaf8f2a282b1da556c2cf1c9b09207.tar.gz

Fixed in #11362 .
Re-open this issue if it still doesn't  work for you.
